### ISSUE TITLE
Report equivocating validator

### ISSUE THREAD

**Issue by cducrest** [trustlines-protocol/blockchain#429]
After live-testing the equivocation with a validator, we should report the equivocation to the validator set to kick out the equivocating validator.
proof should be in https://github.com/trustlines-protocol/blockchain/issues/428

**Comment by jannikluhn** [comment]
The equivocating validator has been reported on the Trustlines chain and has been removed from the validator set. 

I tried to do this using the `report_validator` docker container, but failed (probably because I was confused with docker networks and IP addresses). Installing the `report_validator` tool "manually" did the trick though. 

Link to the transaction: https://explorelaika.trustlines.foundation/tx/0xb0f11dd1fcfd27187da2679acca927900ffeb638d7be156ad79c7d62a5c003ba/internal_transactions

I've confirmed that the validator has indeed been removed by looking at the "validated blocks" page: https://explorelaika.trustlines.foundation/address/0x2aeab6d45754831f88670eb28e86861bb7ba7de2/validations (last block at height 4321697, removal transaction hash at height 4321702). I've also checked the validator set contract directly, calling the `getValidators` function. However, the validator appears to have never been part of the validator set, not even before they were removed.
